src/test/java/software/amazon/documentdb/jdbc/DocumentDbStatementDateTimeTest.java

```diff
@@ -1152,6 +1152,41 @@ void testCurrentTimestampMultipleInstances(final DocumentDbTestEnvironment testE
         }
     }
 
+    /**
+     * Tests select dates with AND condition.
+     *
+     * @throws SQLException occurs if query fails.
+     */
+    @DisplayName("Tests select dates with AND condition.")
+    @ParameterizedTest(name = "testSelectDateWithAnd - [{index}] - {arguments}")
+    @MethodSource({"getTestEnvironments"})
+    void testSelectDateWithAnd(final DocumentDbTestEnvironment testEnvironment) throws SQLException {
+        setTestEnvironment(testEnvironment);
+        final String tableName = "testSelectDateWithAnd";
+        final long dateTime = Instant.parse("2020-02-03T04:05:06.00Z").toEpochMilli();
+        final BsonDocument doc1 = BsonDocument.parse("{\"_id\": 101}");
+        doc1.append("field", new BsonDateTime(dateTime));
+        insertBsonDocuments(tableName, new BsonDocument[]{doc1});
```

Comment:
Can we add a second document where field does not exist and check for that one the result is NULL instead of FALSE?

Reply:
Added NULL row/result